---
result: FAILURE
url: http://manhattan.ci.chef.co/job/chefdk-trigger-ad_hoc/216/
failure_category: code
failure_cause: 'chef_acceptance[top-cookbooks::provision,top-cookbooks::force-destroy]:
  acceptance; yanked gem archive-0.0.6: debian, el, mac_os_x, ubuntu: chefdk-test'
timestamp: 2016-04-22 21:41:54 UTC
duration: 1h11m22s
triggered_by: tyler-ball
active_duration: 1h40m48s
queue_delays: 10s
parameters:
  GIT_REF: unified_workflow_spike
  EXPIRE_CACHE: false
change:
  git_remote: git://github.com/chef/chef-dk.git
  git_commit: 00fefa1366ea3ef72fc97f0bf940af73db4ba6fc
  project: chefdk
  version: 0.13.23+20160422214209
stages:
  chefdk-build/architecture=i386,platform=windows-2008r2,project=chefdk,role=builder:
    result: SUCCESS
    url: http://manhattan.ci.chef.co/job/chefdk-build/architecture=i386,platform=windows-2008r2,project=chefdk,role=builder/316/
    duration: 29m44s
  chefdk-test:
    result: FAILURE
    failure_category: code
    failure_cause: 'chef_acceptance[top-cookbooks::provision,top-cookbooks::force-destroy]:
      acceptance; yanked gem archive-0.0.6: debian, el, mac_os_x, ubuntu'
    url: http://manhattan.ci.chef.co/job/chefdk-test/130/
    duration: 21m15s
    runs:
      acceptance:
        result: FAILURE
        failure_category: code
        failure_cause: chef_acceptance[top-cookbooks::provision,top-cookbooks::force-destroy]
        failed_in:
          chef_acceptance:
          - top-cookbooks::provision
          - top-cookbooks::force-destroy
          step: CHEF-ACCEPTANCE
        url: http://manhattan.ci.chef.co/job/chefdk-test/architecture=x86_64,platform=acceptance,project=chefdk,role=tester/130/
        duration: 1m23s
        logs:
          consoleText:
            527: |
              TOP-COOKBOOKS::PROVISION::[2016-04-22 22:33:17 +0000]

                      ================================================================================
                      Error executing action `sync` on resource 'git[/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/test_run/docker]'
                      ================================================================================

                      Mixlib::ShellOut::ShellCommandFailed
                      ------------------------------------
                      Expected process to exit with [0], but received '128'
                      ---- Begin output of git ls-remote "https://github.com/chef-cookbooks/docker.git" "master*" ----
                      STDOUT:
                      STDERR: fatal: unable to access 'https://github.com/chef-cookbooks/docker.git/': error setting certificate verify locations:
                        CAfile: /opt/chefdk/embedded/ssl/certs
                        CApath: none
                      ---- End output of git ls-remote "https://github.com/chef-cookbooks/docker.git" "master*" ----
                      Ran git ls-remote "https://github.com/chef-cookbooks/docker.git" "master*" returned 128

                      Resource Declaration:
                      ---------------------
                      # In /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/local-mode-cache/cache/cookbooks/acceptance-cookbook/libraries/cookbook_kitchen.rb

                       36:     git repository_root do
                       37:       repository new_resource.repository
                       38:       branch new_resource.branch
                       39:     end
                       40:

                      Compiled Resource:
                      ------------------
                      # Declared in /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/local-mode-cache/cache/cookbooks/acceptance-cookbook/libraries/cookbook_kitchen.rb:36:in `block in <class:CookbookKitchen>'

                      git("/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/test_run/docker") do
                        action [:sync]
                        retries 0
                        retry_delay 2
                        default_guard_interpreter :default
                        destination "/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/test_run/docker"
                        enable_checkout true
                        revision "master"
                        remote "origin"
                        checkout_branch "deploy"
                        declared_type :git
                        cookbook_name "acceptance-cookbook"
                        repository "https://github.com/chef-cookbooks/docker.git"
                      end

                      Platform:
                      ---------
                      x86_64-linux


                    ================================================================================
                    Error executing action `run` on resource 'cookbook_kitchen[converge docker]'
                    ================================================================================

                    Mixlib::ShellOut::ShellCommandFailed
                    ------------------------------------
                    git[/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/test_run/docker] (/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/local-mode-cache/cache/cookbooks/acceptance-cookbook/libraries/cookbook_kitchen.rb line 36) had an error: Mixlib::ShellOut::ShellCommandFailed: Expected process to exit with [0], but received '128'
                    ---- Begin output of git ls-remote "https://github.com/chef-cookbooks/docker.git" "master*" ----
                    STDOUT:
                    STDERR: fatal: unable to access 'https://github.com/chef-cookbooks/docker.git/': error setting certificate verify locations:
                      CAfile: /opt/chefdk/embedded/ssl/certs
                      CApath: none
                    ---- End output of git ls-remote "https://github.com/chef-cookbooks/docker.git" "master*" ----
                    Ran git ls-remote "https://github.com/chef-cookbooks/docker.git" "master*" returned 128

                    Resource Declaration:
                    ---------------------
                    # In /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/local-mode-cache/cache/coo
              TOP-COOKBOOKS::PROVISION::[2016-04-22 22:33:17 +0000] kbooks/acceptance-cookbook/libraries/top_cookbooks.rb
            630: |2


                  ================================================================================
                  Error executing action `run` on resource 'top_cookbooks[converge]'
                  ================================================================================

                  Mixlib::ShellOut::ShellCommandFailed
                  ------------------------------------
                  cookbook_kitchen[converge docker] (/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/local-mode-cache/cache/cookbooks/acceptance-cookbook/libraries/top_cookbooks.rb line 7) had an error: Mixlib::ShellOut::ShellCommandFailed: git[/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/test_run/docker] (/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/local-mode-cache/cache/cookbooks/acceptance-cookbook/libraries/cookbook_kitchen.rb line 36) had an error: Mixlib::ShellOut::ShellCommandFailed: Expected process to exit with [0], but received '128'
                  ---- Begin output of git ls-remote "https://github.com/chef-cookbooks/docker.git" "master*" ----
                  STDOUT:
                  STDERR: fatal: unable to access 'https://github.com/chef-cookbooks/docker.git/': error setting certificate verify locations:
                    CAfile: /opt/chefdk/embedded/ssl/certs
                    CApath: none
                  ---- End output of git ls-remote "https://github.com/chef-cookbooks/docker.git" "master*" ----
                  Ran git ls-remote "https://github.com/chef-cookbooks/docker.git" "master*" returned 128

                  Resource Declaration:
                  ---------------------
                  # In /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/local-mode-cache/cache/cookbooks/acceptance-cookbook/recipes/provision.rb

                    1: top_cookbooks "converge"

                  Compiled Resource:
                  ------------------
                  # Declared in /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/local-mode-cache/cache/cookbooks/acceptance-cookbook/recipes/provision.rb:1:in `from_file'

                  top_cookbooks("converge") do
                    action [:run]
                    updated true
                    updated_by_last_action true
                    retries 0
                    retry_delay 2
                    default_guard_interpreter :default
                    declared_type :top_cookbooks
                    cookbook_name "acceptance-cookbook"
                    recipe_name "provision"
                    command "converge"
                  end

                  Platform:
                  ---------
                  x
              TOP-COOKBOOKS::PROVISION::[2016-04-22 22:33:17 +0000] 86_64-linux
            680: |2

              Running handlers:
              [2016-04-22T22:33:18+00:00] ERROR: Running exception handlers
              Running handlers complete
              [2016-04-22T22:33:18+00:00] ERROR: Exception handlers complete
              Chef Client failed. 1 resources updated in 03 seconds
              TOP-COOKBOOKS::PROVISION::[2016-04-22 22:33:18 +0000] [2016-04-22T22:33:18+00:00] ERROR: Failed to post audit report to server. Saving report to /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/local-mode-cache/cache/failed-audit-data.json
              TOP-COOKBOOKS::PROVISION::[2016-04-22 22:33:18 +0000] [2016-04-22T22:33:18+00:00] FATAL: Stacktrace dumped to /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/local-mode-cache/cache/chef-stacktrace.out
              TOP-COOKBOOKS::PROVISION::[2016-04-22 22:33:18 +0000] [2016-04-22T22:33:18+00:00] FATAL: Please provide the contents of the stacktrace.out file if you file a bug report
              TOP-COOKBOOKS::PROVISION::[2016-04-22 22:33:18 +0000] [2016-04-22T22:33:18+00:00] ERROR: Found 1 errors, they are stored in the backtrace
              TOP-COOKBOOKS::PROVISION::[2016-04-22 22:33:18 +0000] [2016-04-22T22:33:18+00:00] FATAL: Chef::Exceptions::ChildConvergeError: Chef run process exited unsuccessfully (exit code 1)
              CHEF-ACCEPTANCE::[2016-04-22 22:33:18 +0000] Encountered an error running the recipe provision: Expected process to exit with [0], but received '1'
              ---- Begin output of chef-client -z -c /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/config.rb --force-formatter -j /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/dna.json -o acceptance-cookbook::provision --no-color ----
            709: |2
                    * git[/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/test_run/docker] action sync

                      ================================================================================
                      Error executing action `sync` on resource 'git[/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/test_run/docker]'
                      ================================================================================

                      Mixlib::ShellOut::ShellCommandFailed
                      ------------------------------------
                      Expected process to exit with [0], but received '128'
                      ---- Begin output of git ls-remote "https://github.com/chef-cookbooks/docker.git" "master*" ----
                      STDOUT:
                      STDERR: fatal: unable to access 'https://github.com/chef-cookbooks/docker.git/': error setting certificate verify locations:
                        CAfile: /opt/chefdk/embedded/ssl/certs
                        CApath: none
                      ---- End output of git ls-remote "https://github.com/chef-cookbooks/docker.git" "master*" ----
                      Ran git ls-remote "https://github.com/chef-cookbooks/docker.git" "master*" returned 128

                      Resource Declaration:
                      ---------------------
                      # In /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/local-mode-cache/cache/cookbooks/acceptance-cookbook/libraries/cookbook_kitchen.rb

                       36:     git repository_root do
                       37:       repository new_resource.repository
                       38:       branch new_resource.branch
                       39:     end
                       40:

                      Compiled Resource:
                      ------------------
                      # Declared in /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/local-mode-cache/cache/cookbooks/acceptance-cookbook/libraries/cookbook_kitchen.rb:36:in `block in <class:CookbookKitchen>'

                      git("/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/test_run/docker") do
                        action [:sync]
                        retries 0
                        retry_delay 2
                        default_guard_interpreter :default
                        destination "/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/test_run/docker"
                        enable_checkout true
                        revision "master"
                        remote "origin"
                        checkout_branch "deploy"
                        declared_type :git
                        cookbook_name "acceptance-cookbook"
                        repository "https://github.com/chef-cookbooks/docker.git"
                      end

                      Platform:
                      ---------
                      x86_64-linux


                    ================================================================================
                    Error executing action `run` on resource 'cookbook_kitchen[converge docker]'
                    ================================================================================

                    Mixlib::ShellOut::ShellCommandFailed
                    ------------------------------------
                    git[/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/test_run/docker] (/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/local-mode-cache/cache/cookbooks/acceptance-cookbook/libraries/cookbook_kitchen.rb line 36) had an error: Mixlib::ShellOut::ShellCommandFailed: Expected process to exit with [0], but received '128'
                    ---- Begin output of git ls-remote "https://github.com/chef-cookbooks/docker.git" "master*" ----
                    STDOUT:
                    STDERR: fatal: unable to access 'https://github.com/chef-cookbooks/docker.git/': error setting certificate verify locations:
                      CAfile: /opt/chefdk/embedded/ssl/certs
                      CApath: none
                    ---- End output of git ls-remote "https://github.com/chef-cookbooks/docker.git" "master*" ----
                    Ran git ls-remote "https://github.com/chef-cookbooks/docker.git" "master*" returned 128

                    Resource Declaration:
                    ---------------------
                    # In /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/local-mode-cache/cache/cookbooks/acceptance-cookbook/libraries/top_cookbooks.rb

                      7:     cookbook_kitchen "#{command} docker" do
                      8:     end
                      9:

                    Compiled Resource:
                    ------------------
                    # Declared in /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/local-mode-cache/cache/cookbooks/acceptance-cookbook/libraries/top_cookbooks.rb:7:in `block in <class:TopCookbooks>'

                    cookbook_kitchen("converge docker") do
                      action [:run]
                      updated true
                      updated_by_last_action true
                      retries 0
                      retry_delay 2
                      default_guard_interpreter :default
                      declared_type :cookbook_kitchen
                      cookbook_name "acceptance-cookbook"
                      repository "https://github.com/chef-cookbooks/docker.git"
                      test_cookbook "docker"
                      repository_root "/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/test_run/docker"
                      branch "master"
                      command "converge"
                      kitchen_options "-c"
                      kitchen_dir "/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/test_run/docker/"
                      chef_channel "unstable"
                      chef_version "latest"
                      env {"BUNDLE_GEMFILE"=>"/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/Gemfile", "KITCHEN_YAML"=>"/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.kitchen.docker.yml"}
                    end

                    Platform:
                    ---------
                    x86_64-linux


                  ================================================================================
                  Error executing action `run` on resource 'top_cookbooks[converge]'
                  ================================================================================

                  Mixlib::ShellOut::ShellCommandFailed
                  ------------------------------------
                  cookbook_kitchen[converge docker] (/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/local-mode-cache/cache/cookbooks/acceptance-cookbook/libraries/top_cookbooks.rb line 7) had an error: Mixlib::ShellOut::ShellCommandFailed: git[/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/test_run/docker] (/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/local-mode-cache/cache/cookbooks/acceptance-cookbook/libraries/cookbook_kitchen.rb line 36) had an error: Mixlib::ShellOut::ShellCommandFailed: Expected process to exit with [0], but received '128'
                  ---- Begin output of git ls-remote "https://github.com/chef-cookbooks/docker.git" "master*" ----
                  STDOUT:
                  STDERR: fatal: unable to access 'https://github.com/chef-cookbooks/docker.git/': error setting certificate verify locations:
                    CAfile: /opt/chefdk/embedded/ssl/certs
                    CApath: none
                  ---- End output of git ls-remote "https://github.com/chef-cookbooks/docker.git" "master*" ----
                  Ran git ls-remote "https://github.com/chef-cookbooks/docker.git" "master*" returned 128

                  Resource Declaration:
                  ---------------------
                  # In /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/local-mode-cache/cache/cookbooks/acceptance-cookbook/recipes/provision.rb

                    1: top_cookbooks "converge"

                  Compiled Resource:
                  ------------------
                  # Declared in /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/local-mode-cache/cache/cookbooks/acceptance-cookbook/recipes/provision.rb:1:in `from_file'

                  top_cookbooks("converge") do
                    action [:run]
                    updated true
                    updated_by_last_action true
                    retries 0
                    retry_delay 2
                    default_guard_interpreter :default
                    declared_type :top_cookbooks
                    cookbook_name "acceptance-cookbook"
                    recipe_name "provision"
                    command "converge"
                  end

                  Platform:
                  ---------
                  x86_64-linux

              Starting audit phase
            860: |2

              Running handlers:
              [2016-04-22T22:33:18+00:00] ERROR: Running exception handlers
              Running handlers complete
              [2016-04-22T22:33:18+00:00] ERROR: Exception handlers complete
              Chef Client failed. 1 resources updated in 03 seconds
              [2016-04-22T22:33:18+00:00] ERROR: Failed to post audit report to server. Saving report to /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/local-mode-cache/cache/failed-audit-data.json
              [2016-04-22T22:33:18+00:00] FATAL: Stacktrace dumped to /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/local-mode-cache/cache/chef-stacktrace.out
              [2016-04-22T22:33:18+00:00] FATAL: Please provide the contents of the stacktrace.out file if you file a bug report
              [2016-04-22T22:33:18+00:00] ERROR: Found 1 errors, they are stored in the backtrace
              [2016-04-22T22:33:18+00:00] FATAL: Chef::Exceptions::ChildConvergeError: Chef run process exited unsuccessfully (exit code 1)
              STDERR:
              ---- End output of chef-client -z -c /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/config.rb --force-formatter -j /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/dna.json -o acceptance-cookbook::provision --no-color ----
              Ran chef-client -z -c /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/config.rb --force-formatter -j /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/dna.json -o acceptance-cookbook::provision --no-color returned 1
              /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/mixlib-shellout-2.2.6/lib/mixlib/shellout.rb:289:in `invalid!'
              /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/mixlib-shellout-2.2.6/lib/mixlib/shellout.rb:276:in `error!'
              /opt/chefdk/embedded/lib/ruby/gems/2.1.0/bundler/gems/chef-acceptance-49458ec493db/lib/chef-acceptance/chef_runner.rb:40:in `block in run!'
              /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler.rb:219:in `block in with_clean_env'
              /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler.rb:199:in `with_original_env'
              /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler.rb:205:in `with_clean_env'
              /opt/chefdk/embedded/lib/ruby/gems/2.1.0/bundler/gems/chef-acceptance-49458ec493db/lib/chef-acceptance/chef_runner.rb:35:in `run!'
              /opt/chefdk/embedded/lib/ruby/gems/2.1.0/bundler/gems/chef-acceptance-49458ec493db/lib/chef-acceptance/application.rb:144:in `run_command'
              /opt/chefdk/embedded/lib/ruby/gems/2.1.0/bundler/gems/chef-acceptance-49458ec493db/lib/chef-acceptance/application.rb:111:in `block in run_suite'
              /opt/chefdk/embedded/lib/ruby/gems/2.1.0/bundler/gems/chef-acceptance-49458ec493db/lib/chef-acceptance/application.rb:109:in `each'
              /opt/chefdk/embedded/lib/ruby/gems/2.1.0/bundler/gems/chef-acceptance-49458ec493db/lib/chef-acceptance/application.rb:109:in `run_suite'
              /opt/chefdk/embedded/lib/ruby/gems/2.1.0/bundler/gems/chef-acceptance-49458ec493db/lib/chef-acceptance/application.rb:75:in `block (2 levels) in start_worker'
              /opt/chefdk/embedded/lib/ruby/gems/2.1.0/bundler/gems/chef-acceptance-49458ec493db/lib/chef-acceptance/application.rb:73:in `loop'
              /opt/chefdk/embedded/lib/ruby/gems/2.1.0/bundler/gems/chef-acceptance-49458ec493db/lib/chef-acceptance/application.rb:73:in `block in start_worker'
              CHEF-ACCEPTANCE::[2016-04-22 22:33:18 +0000] --force-destroy specified so attempting to run the destroy recipe
              TOP-COOKBOOKS::DESTROY::[2016-04-22 22:33:18 +0000] Initialized [.acceptance_logs/top-cookbooks/destroy.log] logger...
            925: |
              TOP-COOKBOOKS::DESTROY::[2016-04-22 22:33:24 +0000]

                      ================================================================================
                      Error executing action `sync` on resource 'git[/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/test_run/docker]'
                      ================================================================================

                      Mixlib::ShellOut::ShellCommandFailed
                      ------------------------------------
                      Expected process to exit with [0], but received '128'
                      ---- Begin output of git ls-remote "https://github.com/chef-cookbooks/docker.git" "master*" ----
                      STDOUT:
                      STDERR: fatal: unable to access 'https://github.com/chef-cookbooks/docker.git/': error setting certificate verify locations:
                        CAfile: /opt/chefdk/embedded/ssl/certs
                        CApath: none
                      ---- End output of git ls-remote "https://github.com/chef-cookbooks/docker.git" "master*" ----
                      Ran git ls-remote "https://github.com/chef-cookbooks/docker.git" "master*" returned 128

                      Resource Declaration:
                      ---------------------
                      # In /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/local-mode-cache/cache/cookbooks/acceptance-cookbook/libraries/cookbook_kitchen.rb

                       36:     git repository_root do
                       37:       repository new_resource.repository
                       38:       branch new_resource.branch
                       39:     end
                       40:

                      Compiled Resource:
                      ------------------
                      # Declared in /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/local-mode-cache/cache/cookbooks/acceptance-cookbook/libraries/cookbook_kitchen.rb:36:in `block in <class:CookbookKitchen>'

                      git("/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/test_run/docker") do
                        action [:sync]
                        retries 0
                        retry_delay 2
                        default_guard_interpreter :default
                        destination "/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/test_run/docker"
                        enable_checkout true
                        revision "master"
                        remote "origin"
                        checkout_branch "deploy"
                        declared_type :git
                        cookbook_name "acceptance-cookbook"
                        repository "https://github.com/chef-cookbooks/docker.git"
                      end

                      Platform:
                      ---------
                      x86_64-linux


                    ================================================================================
                    Error executing action `run` on resource 'cookbook_kitchen[destroy docker]'
                    ================================================================================

                    Mixlib::ShellOut::ShellCommandFailed
                    ------------------------------------
                    git[/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/test_run/docker] (/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/local-mode-cache/cache/cookbooks/acceptance-cookbook/libraries/cookbook_kitchen.rb line 36) had an error: Mixlib::ShellOut::ShellCommandFailed: Expected process to exit with [0], but received '128'
                    ---- Begin output of git ls-remote "https://github.com/chef-cookbooks/docker.git" "master*" ----
                    STDOUT:
                    STDERR: fatal: unable to access 'https://github.com/chef-cookbooks/docker.git/': error setting certificate verify locations:
                      CAfile: /opt/chefdk/embedded/ssl/certs
                      CApath: none
                    ---- End output of git ls-remote "https://github.com/chef-cookbooks/docker.git" "master*" ----
                    Ran git ls-remote "https://github.com/chef-cookbooks/docker.git" "master*" returned 128

                    Resource Declaration:
                    ---------------------
                    # In /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/local-mode-cache/cache/cook
              TOP-COOKBOOKS::DESTROY::[2016-04-22 22:33:24 +0000] books/acceptance-cookbook/libraries/top_cookbooks.rb
            1026: |2


                  ================================================================================
                  Error executing action `run` on resource 'top_cookbooks[destroy]'
                  ================================================================================

                  Mixlib::ShellOut::ShellCommandFailed
                  ------------------------------------
                  cookbook_kitchen[destroy docker] (/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/local-mode-cache/cache/cookbooks/acceptance-cookbook/libraries/top_cookbooks.rb line 7) had an error: Mixlib::ShellOut::ShellCommandFailed: git[/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/test_run/docker] (/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/local-mode-cache/cache/cookbooks/acceptance-cookbook/libraries/cookbook_kitchen.rb line 36) had an error: Mixlib::ShellOut::ShellCommandFailed: Expected process to exit with [0], but received '128'
                  ---- Begin output of git ls-remote "https://github.com/chef-cookbooks/docker.git" "master*" ----
                  STDOUT:
                  STDERR: fatal: unable to access 'https://github.com/chef-cookbooks/docker.git/': error setting certificate verify locations:
                    CAfile: /opt/chefdk/embedded/ssl/certs
                    CApath: none
                  ---- End output of git ls-remote "https://github.com/chef-cookbooks/docker.git" "master*" ----
                  Ran git ls-remote "https://github.com/chef-cookbooks/docker.git" "master*" returned 128

                  Resource Declaration:
                  ---------------------
                  # In /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/local-mode-cache/cache/cookbooks/acceptance-cookbook/recipes/destroy.rb

                    1: top_cookbooks "destroy"

                  Compiled Resource:
                  ------------------
                  # Declared in /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/local-mode-cache/cache/cookbooks/acceptance-cookbook/recipes/destroy.rb:1:in `from_file'

                  top_cookbooks("destroy") do
                    action [:run]
                    retries 0
                    retry_delay 2
                    default_guard_interpreter :default
                    declared_type :top_cookbooks
                    cookbook_name "acceptance-cookbook"
                    recipe_name "destroy"
                    command "destroy"
                  end

                  Platform:
                  ---------
                  x86_64-linux

              Starting audit phase
              TOP-COOKBOOKS::DESTROY::[2016-04-22 22:33:25 +0000]
            1075: |
              TOP-COOKBOOKS::DESTROY::[2016-04-22 22:33:25 +0000] Running handlers:
              TOP-COOKBOOKS::DESTROY::[2016-04-22 22:33:25 +0000]
              TOP-COOKBOOKS::DESTROY::[2016-04-22 22:33:25 +0000] [2016-04-22T22:33:25+00:00] ERROR: Running exception handlers
              TOP-COOKBOOKS::DESTROY::[2016-04-22 22:33:25 +0000] Running handlers complete
              TOP-COOKBOOKS::DESTROY::[2016-04-22 22:33:25 +0000] [2016-04-22T22:33:25+00:00] ERROR: Exception handlers complete
              TOP-COOKBOOKS::DESTROY::[2016-04-22 22:33:25 +0000] Chef Client failed. 0 resources updated in 03 seconds
              TOP-COOKBOOKS::DESTROY::[2016-04-22 22:33:25 +0000]
              TOP-COOKBOOKS::DESTROY::[2016-04-22 22:33:25 +0000] [2016-04-22T22:33:25+00:00] ERROR: Failed to post audit report to server. Saving report to /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/local-mode-cache/cache/failed-audit-data.json
              TOP-COOKBOOKS::DESTROY::[2016-04-22 22:33:25 +0000] [2016-04-22T22:33:25+00:00] FATAL: Stacktrace dumped to /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.10.13/acceptance/top-cookbooks/.acceptance/acceptance-cookbook/tmp/.chef/local-mode-cache/cache/chef-stacktrace.out
              TOP-COOKBOOKS::DESTROY::[2016-04-22 22:33:25 +0000] [2016-04-22T22:33:25+00:00] FATAL: Please provide the contents of the stacktrace.out file if you file a bug report
              TOP-COOKBOOKS::DESTROY::[2016-04-22 22:33:25 +0000] [2016-04-22T22:33:25+00:00] ERROR: Found 1 errors, they are stored in the backtrace
              TOP-COOKBOOKS::DESTROY::[2016-04-22 22:33:25 +0000] [2016-04-22T22:33:25+00:00] FATAL: Chef::Exceptions::ChildConvergeError: Chef run process exited unsuccessfully (exit code 1)
              CHEF-ACCEPTANCE::[2016-04-22 22:33:25 +0000]
              CHEF-ACCEPTANCE::[2016-04-22 22:33:25 +0000] chef-acceptance run failed
            1092: |
              CHEF-ACCEPTANCE::[2016-04-22 22:33:25 +0000] | top-cookbooks | Total         | 00:00:14 | Y     |
              CHEF-ACCEPTANCE::[2016-04-22 22:33:25 +0000] | Run           | Total         | 00:00:14 | Y     |
              Build step 'Invoke XShell command' marked build as failure
              Archiving artifacts
              Recording fingerprints
        steps:
          total: 1m23s
          before (setup time): 1m9s
          CHEF-ACCEPTANCE: 14s
        chef_acceptance_timing:
        - top-cookbooks:
            provision: 4s
            force-destroy: 4s
            Total: 14s
          Run:
            Total: 14s
      debian-6:
        result: FAILURE
        failure_category: code
        failure_cause: yanked gem archive-0.0.6
        url: http://manhattan.ci.chef.co/job/chefdk-test/architecture=x86_64,platform=debian-6,project=chefdk,role=tester/130/
        duration: 11m50s
        logs:
          consoleText:
            510: |2


              fatal: Not a git repository (or any of the parent directories): .git
              /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:94:in `block in materialize': Could not find archive-0.0.6 in any of the sources (Bundler::GemNotFound)
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `map!'
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `materialize'
            531: |2


              /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:94:in `block in materialize': Could not find logify-0.2.0 in any of the sources (Bundler::GemNotFound)
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `map!'
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `materialize'
            665: |
              Verification of component 'package installation' succeeded.
              Verification of component 'test-kitchen' succeeded.
              Verification of component 'opscode-pushy-client' failed.
              Verification of component 'berkshelf' failed.
              Verification of component 'chefspec' succeeded.
              Verification of component 'chef-provisioning' succeeded.
              Verification of component 'chef-client' failed.
              Verification of component 'chef-dk' failed.
              Build step 'Invoke XShell command' marked build as failure
              Archiving artifacts
              Recording fingerprints
      debian-7:
        result: FAILURE
        failure_category: code
        failure_cause: yanked gem archive-0.0.6
        url: http://manhattan.ci.chef.co/job/chefdk-test/architecture=x86_64,platform=debian-7,project=chefdk,role=tester/130/
        duration: 8m3s
        logs:
          consoleText:
            495: |2


              fatal: Not a git repository (or any of the parent directories): .git
              /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:94:in `block in materialize': Could not find archive-0.0.6 in any of the sources (Bundler::GemNotFound)
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `map!'
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `materialize'
            516: |2


              /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:94:in `block in materialize': Could not find logify-0.2.0 in any of the sources (Bundler::GemNotFound)
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `map!'
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `materialize'
            650: |
              Verification of component 'package installation' succeeded.
              Verification of component 'test-kitchen' succeeded.
              Verification of component 'opscode-pushy-client' failed.
              Verification of component 'berkshelf' failed.
              Verification of component 'chefspec' succeeded.
              Verification of component 'chef-provisioning' succeeded.
              Verification of component 'chef-client' failed.
              Verification of component 'chef-dk' failed.
              Build step 'Invoke XShell command' marked build as failure
              Archiving artifacts
              Recording fingerprints
      debian-8:
        result: FAILURE
        failure_category: code
        failure_cause: yanked gem archive-0.0.6
        url: http://manhattan.ci.chef.co/job/chefdk-test/architecture=x86_64,platform=debian-8,project=chefdk,role=tester/130/
        duration: 9m18s
        logs:
          consoleText:
            493: |2


              fatal: Not a git repository (or any of the parent directories): .git
              /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:94:in `block in materialize': Could not find archive-0.0.6 in any of the sources (Bundler::GemNotFound)
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `map!'
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `materialize'
            514: |2


              /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:94:in `block in materialize': Could not find logify-0.2.0 in any of the sources (Bundler::GemNotFound)
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `map!'
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `materialize'
            648: |
              Verification of component 'package installation' succeeded.
              Verification of component 'test-kitchen' succeeded.
              Verification of component 'opscode-pushy-client' failed.
              Verification of component 'berkshelf' failed.
              Verification of component 'chefspec' succeeded.
              Verification of component 'chef-provisioning' succeeded.
              Verification of component 'chef-client' failed.
              Verification of component 'chef-dk' failed.
              Build step 'Invoke XShell command' marked build as failure
              Archiving artifacts
              Recording fingerprints
      el-6:
        result: FAILURE
        failure_category: code
        failure_cause: yanked gem archive-0.0.6
        url: http://manhattan.ci.chef.co/job/chefdk-test/architecture=x86_64,platform=el-6,project=chefdk,role=tester/130/
        duration: 21m14s
        logs:
          consoleText:
            482: |2


              fatal: Not a git repository (or any of the parent directories): .git
              /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:94:in `block in materialize': Could not find archive-0.0.6 in any of the sources (Bundler::GemNotFound)
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `map!'
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `materialize'
            503: |2


              /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:94:in `block in materialize': Could not find logify-0.2.0 in any of the sources (Bundler::GemNotFound)
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `map!'
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `materialize'
            653: |
              Verification of component 'package installation' succeeded.
              Verification of component 'test-kitchen' succeeded.
              Verification of component 'opscode-pushy-client' failed.
              Verification of component 'berkshelf' failed.
              Verification of component 'chefspec' succeeded.
              Verification of component 'chef-client' failed.
              Verification of component 'chef-provisioning' succeeded.
              Verification of component 'chef-dk' failed.
              Build step 'Invoke XShell command' marked build as failure
              Archiving artifacts
              Recording fingerprints
      el-7:
        result: FAILURE
        failure_category: code
        failure_cause: yanked gem archive-0.0.6
        url: http://manhattan.ci.chef.co/job/chefdk-test/architecture=x86_64,platform=el-7,project=chefdk,role=tester/130/
        duration: 7m14s
        logs:
          consoleText:
            489: |2


              fatal: Not a git repository (or any of the parent directories): .git
              /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:94:in `block in materialize': Could not find archive-0.0.6 in any of the sources (Bundler::GemNotFound)
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `map!'
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `materialize'
            510: |2


              /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:94:in `block in materialize': Could not find logify-0.2.0 in any of the sources (Bundler::GemNotFound)
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `map!'
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `materialize'
            660: |
              Verification of component 'package installation' succeeded.
              Verification of component 'test-kitchen' succeeded.
              Verification of component 'opscode-pushy-client' failed.
              Verification of component 'berkshelf' failed.
              Verification of component 'chef-provisioning' succeeded.
              Verification of component 'chefspec' succeeded.
              Verification of component 'chef-client' failed.
              Verification of component 'chef-dk' failed.
              Build step 'Invoke XShell command' marked build as failure
              Archiving artifacts
              Recording fingerprints
      mac_os_x-10.10:
        result: FAILURE
        failure_category: code
        failure_cause: yanked gem archive-0.0.6
        url: http://manhattan.ci.chef.co/job/chefdk-test/architecture=x86_64,platform=mac_os_x-10.10,project=chefdk,role=tester/130/
        duration: 5m16s
        logs:
          consoleText:
            389: |
              /dev/disk1            GUID_partition_scheme
              /dev/disk1s1          Apple_HFS                        /Volumes/chefdk
              find: /Volumes/chefdk/.Trashes: Permission denied
              installer: Package name is Chef Development Kit
              installer: Installing at base path /
            444: |
              For non-Windows platforms, make sure libzmq is located in this search path:
              ["/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/ffi-rzmq-core-1.0.4/lib/ffi-rzmq-core/../../ext/libzmq.dylib", "/opt/chefdk/bin/libzmq.dylib", "/opt/chefdk/embedded/bin/libzmq.dylib", "/opt/chefdk/bin/libzmq.dylib", "/opt/chefdk/bin/libzmq.dylib", "/opt/languages/ruby/2.1.5/bin/libzmq.dylib", "/usr/local/bin/libzmq.dylib", "/libzmq.dylib", "/usr/local/sbin/libzmq.dylib", "/usr/local/bin/libzmq.dylib", "/usr/sbin/libzmq.dylib", "/usr/bin/libzmq.dylib", "/sbin/libzmq.dylib", "/bin/libzmq.dylib", "/usr/local/lib/libzmq.dylib", "/opt/local/lib/libzmq.dylib", "/usr/local/lib/libzmq.dylib", "/usr/lib64/libzmq.dylib"]
              /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/ffi-rzmq-core-1.0.4/lib/ffi-rzmq-core/libzmq.rb:54:in `rescue in <module:LibZMQ>': The libzmq library (or DLL) could not be loaded (LoadError)
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/ffi-rzmq-core-1.0.4/lib/ffi-rzmq-core/libzmq.rb:9:in `<module:LibZMQ>'
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/ffi-rzmq-core-1.0.4/lib/ffi-rzmq-core/libzmq.rb:6:in `<top (required)>'
            537: |2


              fatal: Not a git repository (or any of the parent directories): .git
              /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:94:in `block in materialize': Could not find archive-0.0.6 in any of the sources (Bundler::GemNotFound)
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `map!'
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `materialize'
            558: |2


              /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:94:in `block in materialize': Could not find logify-0.2.0 in any of the sources (Bundler::GemNotFound)
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `map!'
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `materialize'
            692: |
              Verification of component 'package installation' succeeded.
              Verification of component 'test-kitchen' succeeded.
              Verification of component 'opscode-pushy-client' failed.
              Verification of component 'berkshelf' failed.
              Verification of component 'chefspec' succeeded.
              Verification of component 'chef-provisioning' succeeded.
              Verification of component 'chef-client' failed.
              Verification of component 'chef-dk' failed.
              Build step 'Invoke XShell command' marked build as failure
              Archiving artifacts
              Recording fingerprints
      mac_os_x-10.11:
        result: FAILURE
        failure_category: code
        failure_cause: yanked gem archive-0.0.6
        url: http://manhattan.ci.chef.co/job/chefdk-test/architecture=x86_64,platform=mac_os_x-10.11,project=chefdk,role=tester/130/
        duration: 3m55s
        logs:
          consoleText:
            389: |
              /dev/disk1            GUID_partition_scheme
              /dev/disk1s1          Apple_HFS                        /Volumes/chefdk
              find: /Volumes/chefdk/.Trashes: Permission denied
              installer: Package name is Chef Development Kit
              installer: Installing at base path /
            444: |
              For non-Windows platforms, make sure libzmq is located in this search path:
              ["/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/ffi-rzmq-core-1.0.4/lib/ffi-rzmq-core/../../ext/libzmq.dylib", "/opt/chefdk/bin/libzmq.dylib", "/opt/chefdk/embedded/bin/libzmq.dylib", "/opt/chefdk/bin/libzmq.dylib", "/opt/chefdk/bin/libzmq.dylib", "/opt/languages/ruby/2.1.5/bin/libzmq.dylib", "/usr/local/bin/libzmq.dylib", "/libzmq.dylib", "/usr/local/sbin/libzmq.dylib", "/usr/local/bin/libzmq.dylib", "/usr/sbin/libzmq.dylib", "/usr/bin/libzmq.dylib", "/sbin/libzmq.dylib", "/bin/libzmq.dylib", "/usr/local/lib/libzmq.dylib", "/opt/local/lib/libzmq.dylib", "/usr/local/lib/libzmq.dylib", "/usr/lib64/libzmq.dylib"]
              /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/ffi-rzmq-core-1.0.4/lib/ffi-rzmq-core/libzmq.rb:54:in `rescue in <module:LibZMQ>': The libzmq library (or DLL) could not be loaded (LoadError)
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/ffi-rzmq-core-1.0.4/lib/ffi-rzmq-core/libzmq.rb:9:in `<module:LibZMQ>'
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/ffi-rzmq-core-1.0.4/lib/ffi-rzmq-core/libzmq.rb:6:in `<top (required)>'
            537: |2


              fatal: Not a git repository (or any of the parent directories): .git
              /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:94:in `block in materialize': Could not find archive-0.0.6 in any of the sources (Bundler::GemNotFound)
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `map!'
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `materialize'
            558: |2


              /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:94:in `block in materialize': Could not find logify-0.2.0 in any of the sources (Bundler::GemNotFound)
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `map!'
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `materialize'
            692: |
              Verification of component 'package installation' succeeded.
              Verification of component 'test-kitchen' succeeded.
              Verification of component 'opscode-pushy-client' failed.
              Verification of component 'berkshelf' failed.
              Verification of component 'chef-provisioning' succeeded.
              Verification of component 'chefspec' succeeded.
              Verification of component 'chef-client' failed.
              Verification of component 'chef-dk' failed.
              Build step 'Invoke XShell command' marked build as failure
              Archiving artifacts
              Recording fingerprints
      mac_os_x-10.9:
        result: FAILURE
        failure_category: code
        failure_cause: yanked gem archive-0.0.6
        url: http://manhattan.ci.chef.co/job/chefdk-test/architecture=x86_64,platform=mac_os_x-10.9,project=chefdk,role=tester/130/
        duration: 4m58s
        logs:
          consoleText:
            389: |
              /dev/disk1            GUID_partition_scheme
              /dev/disk1s1          Apple_HFS                        /Volumes/chefdk
              find: /Volumes/chefdk/.Trashes: Permission denied
              installer: Package name is Chef Development Kit
              installer: Installing at base path /
            444: |
              For non-Windows platforms, make sure libzmq is located in this search path:
              ["/opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/ffi-rzmq-core-1.0.4/lib/ffi-rzmq-core/../../ext/libzmq.dylib", "/opt/chefdk/bin/libzmq.dylib", "/opt/chefdk/embedded/bin/libzmq.dylib", "/opt/chefdk/bin/libzmq.dylib", "/opt/chefdk/bin/libzmq.dylib", "/opt/languages/ruby/2.1.5/bin/libzmq.dylib", "/usr/local/bin/libzmq.dylib", "/libzmq.dylib", "/usr/local/sbin/libzmq.dylib", "/usr/local/bin/libzmq.dylib", "/usr/sbin/libzmq.dylib", "/usr/bin/libzmq.dylib", "/sbin/libzmq.dylib", "/bin/libzmq.dylib", "/usr/local/lib/libzmq.dylib", "/opt/local/lib/libzmq.dylib", "/usr/local/lib/libzmq.dylib", "/usr/lib64/libzmq.dylib"]
              /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/ffi-rzmq-core-1.0.4/lib/ffi-rzmq-core/libzmq.rb:54:in `rescue in <module:LibZMQ>': The libzmq library (or DLL) could not be loaded (LoadError)
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/ffi-rzmq-core-1.0.4/lib/ffi-rzmq-core/libzmq.rb:9:in `<module:LibZMQ>'
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/ffi-rzmq-core-1.0.4/lib/ffi-rzmq-core/libzmq.rb:6:in `<top (required)>'
            537: |2


              fatal: Not a git repository (or any of the parent directories): .git
              /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:94:in `block in materialize': Could not find archive-0.0.6 in any of the sources (Bundler::GemNotFound)
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `map!'
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `materialize'
            558: |2


              /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:94:in `block in materialize': Could not find logify-0.2.0 in any of the sources (Bundler::GemNotFound)
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `map!'
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `materialize'
            692: |
              Verification of component 'chef-sugar' succeeded.
              Verification of component 'test-kitchen' succeeded.
              Verification of component 'opscode-pushy-client' failed.
              Verification of component 'berkshelf' failed.
              Verification of component 'chef-provisioning' succeeded.
              Verification of component 'chefspec' succeeded.
              Verification of component 'chef-client' failed.
              Verification of component 'chef-dk' failed.
              Build step 'Invoke XShell command' marked build as failure
              Archiving artifacts
              Recording fingerprints
      ubuntu-12.04:
        result: FAILURE
        failure_category: code
        failure_cause: yanked gem archive-0.0.6
        url: http://manhattan.ci.chef.co/job/chefdk-test/architecture=x86_64,platform=ubuntu-12.04,project=chefdk,role=tester/130/
        duration: 7m28s
        logs:
          consoleText:
            495: |2


              fatal: Not a git repository (or any of the parent directories): .git
              /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:94:in `block in materialize': Could not find archive-0.0.6 in any of the sources (Bundler::GemNotFound)
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `map!'
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `materialize'
            516: |2


              /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:94:in `block in materialize': Could not find logify-0.2.0 in any of the sources (Bundler::GemNotFound)
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `map!'
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `materialize'
            650: |
              Verification of component 'package installation' succeeded.
              Verification of component 'test-kitchen' succeeded.
              Verification of component 'opscode-pushy-client' failed.
              Verification of component 'berkshelf' failed.
              Verification of component 'chef-client' failed.
              Verification of component 'chef-provisioning' succeeded.
              Verification of component 'chefspec' succeeded.
              Verification of component 'chef-dk' failed.
              Build step 'Invoke XShell command' marked build as failure
              Archiving artifacts
              Recording fingerprints
      ubuntu-14.04:
        result: FAILURE
        failure_category: code
        failure_cause: yanked gem archive-0.0.6
        url: http://manhattan.ci.chef.co/job/chefdk-test/architecture=x86_64,platform=ubuntu-14.04,project=chefdk,role=tester/130/
        duration: 11m8s
        logs:
          consoleText:
            497: |2


              fatal: Not a git repository (or any of the parent directories): .git
              /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:94:in `block in materialize': Could not find archive-0.0.6 in any of the sources (Bundler::GemNotFound)
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `map!'
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `materialize'
            518: |2


              /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:94:in `block in materialize': Could not find logify-0.2.0 in any of the sources (Bundler::GemNotFound)
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `map!'
                from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.11.2/lib/bundler/spec_set.rb:87:in `materialize'
            652: |
              Verification of component 'package installation' succeeded.
              Verification of component 'test-kitchen' succeeded.
              Verification of component 'opscode-pushy-client' failed.
              Verification of component 'berkshelf' failed.
              Verification of component 'chefspec' succeeded.
              Verification of component 'chef-provisioning' succeeded.
              Verification of component 'chef-client' failed.
              Verification of component 'chef-dk' failed.
              Build step 'Invoke XShell command' marked build as failure
              Archiving artifacts
              Recording fingerprints
      windows-2008r2-i386:
        result: SUCCESS
        url: http://manhattan.ci.chef.co/job/chefdk-test/architecture=i386,platform=windows-2008r2,project=chefdk,role=tester/130/
        duration: 9m39s
  chefdk-build:
    result: SUCCESS
    url: http://manhattan.ci.chef.co/job/chefdk-build/316/
    duration: 49m43s
    queue_delay: 10s
    runs:
      windows-2008r2-i386:
        result: SUCCESS
        url: http://manhattan.ci.chef.co/job/chefdk-build/architecture=i386,platform=windows-2008r2,project=chefdk,role=builder/316/
        duration: 29m44s
      debian-6:
        result: SUCCESS
        url: http://manhattan.ci.chef.co/job/chefdk-build/architecture=x86_64,platform=debian-6,project=chefdk,role=builder/316/
        duration: 42m17s
      el-6:
        result: SUCCESS
        url: http://manhattan.ci.chef.co/job/chefdk-build/architecture=x86_64,platform=el-6,project=chefdk,role=builder/316/
        duration: 49m38s
      el-7:
        result: SUCCESS
        url: http://manhattan.ci.chef.co/job/chefdk-build/architecture=x86_64,platform=el-7,project=chefdk,role=builder/316/
        duration: 38m52s
      mac_os_x-10.9:
        result: SUCCESS
        url: http://manhattan.ci.chef.co/job/chefdk-build/architecture=x86_64,platform=mac_os_x-10.9,project=chefdk,role=builder/316/
        duration: 16m25s
      ubuntu-12.04:
        result: SUCCESS
        url: http://manhattan.ci.chef.co/job/chefdk-build/architecture=x86_64,platform=ubuntu-12.04,project=chefdk,role=builder/316/
        duration: 41m30s
  chefdk-trigger-ad_hoc:
    result: SUCCESS
    url: http://manhattan.ci.chef.co/job/chefdk-trigger-ad_hoc/216/
    duration: 4s
successful_logs_analyzed: false
